# 比赛结果

Header header

# 比赛结果
# 平局
uint8 RESULT_DRAW = 0
# 红方胜利
uint8 RESULT_RED = 1
# 蓝方胜利
uint8 RESULT_BLUE = 2
uint8 result